ui/components/ui/update-nickname-popover/update-nickname-popover.stories.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import { action } from '@storybook/addon-actions';
+import Button from '../button';
+import UpdateNicknamePopover from '.';
+
+export default {
+  title: 'UpdateNickname',
+  id: __filename,
+};
+
+export const AddNickname = () => {
+  const [showPopover, setShowPopover] = useState(false);
+  return (
+    <div style={{ width: '600px' }}>
+      <Button onClick={() => setShowPopover(true)}>
+        Open Add Nickname Popover
+      </Button>
+      {showPopover && (
+        <UpdateNicknamePopover
+          address="0x0011244f50ff4"
+          onClose={() => action(`Close Update Nickname Popover`)()}
+        />
+      )}
+    </div>
+  );
+};
+
+export const UpdateNickname = () => {
+  const [showPopover, setShowPopover] = useState(false);
+  return (
+    <div style={{ width: '600px' }}>
+      <Button onClick={() => setShowPopover(true)}>
+        Open Update Nickname Popover
+      </Button>
+      {showPopover && (
+        <UpdateNicknamePopover
+          address="0x0011244f50ff4"
```

Comment:
You can import { text } from '@storybook/addon-knobs'; and then change this to address={text('0x0011244f50ff4')} This will allow the text to be editable in the storybook component

Same for the nickname below

Reply:
Done.